### PR TITLE
Kiwi 1414 qa

### PR DESCRIPTION
**Pull request by Z-Mohamoud**
### What changed

Added test in the e2eHappypath

### Why did it change

To validate Include Additional Claims in VC

- [KIWI-1414](https://govukverify.atlassian.net/browse/KIWI-1414)

## Checklists

[KIWI-1414]: https://govukverify.atlassian.net/browse/KIWI-1414?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MohamedHassan101 — APPROVED** [review]
Nice

File: deploy/template.yaml [hunk 1]
```diff
@@ -4225,4 +4225,13 @@ Outputs:
...
\ No newline at end of file
+    Value: !FindInMap [EnvironmentVariables, !Ref Environment, TESTHARNESSURL]
+  VcSigningKeyId:
+    Condition: IsNotProdLikeEnvironment
+    Description: "Signing Key used to sign VC"
+    Value:
+      Fn::ImportValue: !Sub "${L2KMSStackName}-vc-signing-key-id"
+  DNSSuffix:
+    Condition: IsNotProdLikeEnvironment
+    Description: "F2F DNS Suffix"
```
Comment (Caitlin-cooling) [inline deploy/template.yaml:4236]: CIC not F2F

File: src/tests/api/e2eHappyPath.test.ts [hunk 2]
```diff
@@ -15,7 +15,7 @@ import {
 import { getTxmaEventsFromTestHarness, validateTxMAEventData } from "./ApiUtils";
 
 describe("E2E Happy Path Tests", () => {
-	it.each([
+	it.only.each([
```
[the remark this answers is not in the document]
Comment (Caitlin-cooling) [reply src/tests/api/e2eHappyPath.test.ts:18]: Also just want to confirm that we've run the whole suite and are sure all the tests are passing?

File: deploy/template.yaml (hunk 1 above)
Comment (Z-Mohamoud) [reply deploy/template.yaml:4236]: Updated

File: src/tests/api/e2eHappyPath.test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (Z-Mohamoud) [reply src/tests/api/e2eHappyPath.test.ts:18]: Removed and confirmed that the full suite has passed.

**Review by Caitlin-cooling — APPROVED** [review]

**Pull request merged**